crates/biome_grit_formatter/tests/specs/grit/options.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Wonder if this file can be deleted since it contains no content

Reply:
Thanks for calling this out! I threw this in there because the formatter options weren't being read in. I thought I had already added all of the needed configuration for the defaults, but I was missing a few extra functions which I added in for this PR.